src/Object/Contact.php

```diff
@@ -143,14 +134,13 @@ public static function markForArchival(array $contact)
 	 *
 	 * @see Contact::markForArchival()
 	 *
-	 * @param array $contact
+	 * @param array $contact contact to be unmarked for archival
 	 * @return null
 	 */
 	public static function unmarkForArchival(array $contact)
 	{
-		$r = q(
-			"SELECT `term-date` FROM `contact` WHERE `id` = %d AND (`term-date` > '%s' OR `archive`)", intval($contact['id']), dbesc('1000-00-00 00:00:00')
-		);
+		$condition = array('`id` => ? AND (`term-date` > ? OR `archive`)', $contact[`id`], NULL_DATE);
```

Comment:
I'm asking myself if we have to keep using the date '1000-00-00' because we marked the contacts in the past with that date. But on the other hand it seems better to make it similar everywhere in the code.

Since this shouldn't break things, we should just try it with ```NULL_DATE``` like you did.

Reply:
I already verified `term-date` is nulled with `NULL_DATE`, so it will work.